### ISSUE TITLE
Add global path for logs

### ISSUE THREAD

**Issue by SorinGFS** [valentinxxx/nginxconfig.io#405]
### Feature description
Allow user to customize the global path for logs:

![Untitled](https://user-images.githubusercontent.com/25417015/201736458-6648a1df-84ab-4698-b081-f3248a3fdbac.png)

**Comment by MattIPv4** [comment]
Please can you specify details on how you expect this to interact with the user-customisable per-domain log paths we now have?

How would this work in a way which isn't confusing UX for the user? If they edit the per-domain path, and then edit this new setting, what happens? If they leave the per-domain path as the default, and then edit this new setting, what happens in the per-domain UI? etc.

**Comment by SorinGFS** [comment]
> If they edit the per-domain path, and then edit this new setting, what happens? If they leave the per-domain path as the default, and then edit this new setting, what happens in the per-domain UI? etc.

This new global variable should be used to construct the default path (the one that appears in logging tab): `${nginx_logs_base_path} + access.log` and `${nginx_logs_base_path} + error.log`. This is usefull because user doesn't need to cusomize every domain in case he uses another path then default. If the user chooses to further customize the path per domain this would be the his starting value, 

If we have this new modification in place, then we would be able to inject `${server_name}` variable into the path without problem. Let's not talk about this for the moment, I think the feature I propose is usefull on its own.

**Comment by MattIPv4** [comment]
>  If the user chooses to further customize the path per domain this would be his starting value,

Right, but once the user has further customized the path, what is the expected UX of this global setting? Does it become disabled permanently because there is a custom path being used? Or does it remain active? And if it does remain active, what happens if it is changed again, does it reset the customised value?

**Comment by SorinGFS** [comment]
> Does it become disabled permanently because there is a custom path being used? Or does it remain active? And if it does remain active, what happens if it is changed again, does it reset the customised value?

It should remain active for the rest of the non cusomized domains. If you concern what happens if user cusomizes the per domain value then goes back and cusomizes the global value... the solution can be a check if the customized path is still equal with global value. In this case changing the global value should not affect the customized value.

**Comment by MattIPv4** [comment]
I am still of the belief that this will be very confusing UX for the user, with two settings in two very different parts of the app that influence each other and the output config.

As such, I don't think this is something that should be worked on for now, but will leave this issue open so others in the community can express their support if they feel this change would be useful and outweigh the potential confusion caused by the UX.

**Comment by SorinGFS** [comment]
@MattIPv4 

How about having the path in the logging tab with the first part (base path) taken from `nginx` global path and greyed out, and allowing the user to customize only the rest of the path? I think no one uses different base paths for different domains...
